src/config/dark-sites.config

```diff
@@ -1040,38 +958,35 @@ save.tf
 sb.ltn.fi
 schildi.chat
 science-news.co
-scottgames.com
-screfy.com
 scriptkit.com
 seaofthieves.fandom.com
 search.biboumail.fr
 search.brave.com
-search.dr460nf1r3.org
 search.nebulacentre.net
 secrethitler.io
 secure.eveonline.com
 senkognito.com
 senkuro.com
-senpai.gg
 septatrix.github.io/prefers-color-scheme-test/
 serebii.net
 settings.gg
 seximal.net
 seyi.dev
+sharetextures.com
 sharkiller.ddns.net
 shatteredpixel.com
 sheet.host
 shellshock.io
 sherlock-project.github.io
 shockbs.is-a.dev
+shockbs.pro
 showtimeanytime.com
 shpposter.club
-shrirambalaji.dev
-shsh.host
+shrirangkahale.com
```

Comment:
Site not always dark - it has button to change dark/light mode:
![obraz](https://github.com/user-attachments/assets/db11c034-03c8-4c91-9e49-b3f2619482fe)

To remove!

Reply:
Not sure why this was set as resolved.